frontend/src/components/customer/ProfileDetailsCard.tsx/ProfileDetailsCard.tsx

```diff
@@ -0,0 +1,480 @@
+                                        <Grid item>
+                                            <input
+                                                type="file"
+                                                accept="image/*"
+                                                id="profile-picture-upload"
+                                                style={{ display: "none" }}
+                                                onChange={handleFileUpload}
+                                            />
+                                            <label htmlFor="profile-picture-upload">
+                                                <Button
+                                                    variant="contained"
+                                                    component="span"
+                                                >
+                                                    Upload
+                                                </Button>
+                                            </label>
+                                        </Grid>
+                                        <Grid item>
+                                            {profilePicture && (
+                                                <Button
+                                                    color="error"
+                                                    variant="contained"
+                                                    onClick={
+                                                        handleDeletePicture
+                                                    }
+                                                >
+                                                    Delete
+                                                </Button>
+                                            )}
+                                        </Grid>
+                                    </Grid>
+                                </Grid>
+                            </Grid>
+                            <Grid item xs={8}>
+                                <Grid
+                                    container
+                                    direction="row"
+                                    alignItems="flex-start"
+                                    gap={3}
+                                >
+                                    <Grid item xs={5.5}>
+                                        <Grid
+                                            container
+                                            direction="column"
+                                            gap={3}
+                                            justifyContent="flex-start"
+                                            alignItems="flex-start"
+                                            alignContent="flex-start"
+                                        >
+                                            <Grid item>
+                                                <TextField
```

Comment:
No need id, type is `text`. Read the API documentations for TextField

Reply:
And btw, I have already created a `TextField` component for our app, please import the one under `components/common/form/TextField.tsx`, not the one by MUI